### PR TITLE
Testing http_status beacon on ovs-rc

### PR DESCRIPTION
**Pull request by shaidar**

### CODE REVIEW

File: pillar/beacons/http_status_odl_video_service.sls [hunk 1]
```diff
@@ -0,0 +1,10 @@
+beacons:
+  http_status:
+    - sites:
+        odl-video-rc-apps:
+          url: "https://video-rc.odl.mit.edu/status?token=rc-apps"
+          json_response:
+            - path: 'certificate:status'
+              value: up
```
Comment (blarghmatey) [inline pillar/beacons/http_status_odl_video_service.sls]: Is this missing the comparison or does it default to `==` if there is no setting?

**Comment by blarghmatey** [comment]
Do we need to set up a reactor?

File: pillar/beacons/http_status_odl_video_service.sls (hunk 1 above)
Comment (shaidar) [reply pillar/beacons/http_status_odl_video_service.sls]: Just added that. Forgot about it.

**Comment by shaidar** [comment]
Wanted to initially just make sure the extension works as expected and was then going to add all the pillar and reactor stuff.

**Comment by blarghmatey** [comment]
:+1:

**Pull request merged**